test/util/vault/util.go

```diff
@@ -63,12 +64,17 @@ func NewVaultInitializer(container, rootMount, intermediateMount, role string) (
 
 	client.SetToken(vaultToken)
 
+	if authPath == "" {
+		return nil, fmt.Errorf("Error authPath must be set")
+	}
```

Comment:
authPath is an optional field, we needn't require it be set in our fixtures (this causes disparity between tests and docs 😄)